src/mumble/AudioInput.cpp

```diff
@@ -384,6 +384,14 @@ void AudioInput::addMic(const void *data, unsigned int nsamp) {
 					if (qlEchoFrames.isEmpty()) {
 						iJitterSeq = 0;
 						iMinBuffered = 1000;
+					} else if (qlEchoFrames.count() >= 100) {
```

Comment:
Didn't think of it as a refactor. More like providing it und using it just here so we can use it going forward. Equivalent to ```SAMPLE_RATE```. I wouldn't bother to derive it as it doesn't change (at least as far as I know).